Cargo.toml

```diff
@@ -25,11 +25,12 @@ freetype-rs = "0.26"
 pkg-config = "0.3"
 
 [target.'cfg(target_os = "macos")'.dependencies]
-cocoa = "0.24"
+cacao = "0.2.1"
 core-foundation = "0.9"
 core-text = "19"
 core-graphics = "0.22"
 core-foundation-sys = "0.8"
+lazy_static = "1.4.0"
```

Comment:
Could you use `once_cell` instead?

Reply:
Sure. I actually did it with `once_cell` originally but thought `lazy_static` was more common. I can change it back.

Reply:
`once_cell` is the Api that will get eventually into `std`, so once it'll be there we can safely drop the dependency on external crates entirely.

Reply:
Fixed.